10.yaml

```
@@ -0,0 +1,49 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: q10
+  labels:
+    question.kontest.amsy.dev: "10"
+data:
+  question: |-
+    Kubernetesで永続化領域の要求を行うリソースが、PersistentVolumeClaimです。永続化領域の要求が発行されるとKube-Schedulerが保持しているPersistentVolumeから最適なVolumeを割り当てます。
```

```suggestion
    Kubernetesにおいて、永続化領域の要求を行うリソースがPersistentVolumeClaim(PVC)です。PVCが作成されると、利用可能なPersistentVolume(PV)の中から最適なリソースがPodに割り当てられます。
```

`Kube-Schedulerが保持している` だと、kube-schedulerがPVを所有しているような日本語に見えるのと、問題的には表記する必要がなさそうだったので消してみました。